### PR TITLE
Activate host net containers on kubernetes.

### PR DESCRIPTION
**Pull request by varks**
This PR activates pods with property hostNetwork set to true in K8s pod specifications. Such pods are activated as single LinuxProcessPU. All the other container in the pods are added to the same cgroup as of the pause container.

### CODE REVIEW

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "kube_host",
    "component": "trireme-lib",
    "pr-id": "689",
    "commit-sha": "e5d8012c979f4f8e29ea2739f967b10735264724"
  },
  {
    "project": "kube_host",
    "component": "enforcerd",
    "pr-id": "915",
    "commit-sha": "be80bfda5de58f5ef5e28db3c37d1edca74c722b"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "kube_host",
    "component": "trireme-lib",
    "pr-id": "689",
    "commit-sha": "e5d8012c979f4f8e29ea2739f967b10735264724"
  },
  {
    "project": "kube_host",
    "component": "enforcerd",
    "pr-id": "915",
    "commit-sha": "f9262c912028a4f3b51cb6a83aced2a058ae03c8"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "kube_host",
    "component": "trireme-lib",
    "pr-id": "689",
    "commit-sha": "ec29f396986b550996c54fc41c03c16743aedcec"
  },
  {
    "project": "kube_host",
    "component": "enforcerd",
    "pr-id": "915",
    "commit-sha": "dee4c2026f0dfb72b344556eaf9d7b2deb466b10"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "kube_host",
    "component": "trireme-lib",
    "pr-id": "689",
    "commit-sha": "ec29f396986b550996c54fc41c03c16743aedcec"
  },
  {
    "project": "kube_host",
    "component": "enforcerd",
    "pr-id": "915",
    "commit-sha": "4130121d100c04e01440c0055f98aac3496af04a"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "kube_host",
    "component": "trireme-lib",
    "pr-id": "689",
    "commit-sha": "e826b345c200ebd2930e3b94ec024cea845bae41"
  },
  {
    "project": "kube_host",
    "component": "enforcerd",
    "pr-id": "915",
    "commit-sha": "9a9ad104135a007bb0b23b39e09e617559073963"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "kube_host",
    "component": "trireme-lib",
    "pr-id": "689",
    "commit-sha": "59714caa53d031bfc281f6982f29e8ca8a1c71ae"
  },
  {
    "project": "kube_host",
    "component": "enforcerd",
    "pr-id": "915",
    "commit-sha": "9a9ad104135a007bb0b23b39e09e617559073963"
  }
]
```

File: monitor/internal/docker/monitor.go [hunk 1]
```diff
@@ -298,40 +312,86 @@ func (d *DockerMonitor) resyncContainers(ctx context.Context, containers []types
 		}
 
 	}
+
```
Comment (dstiliadis) [inline monitor/internal/docker/monitor.go]: Can we make this a function  and call it twice with the right parameters?

**Review by dstiliadis — CHANGES_REQUESTED** [review]
Let's combine the statements in a single function and call twice.

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "kube_host",
    "component": "trireme-lib",
    "pr-id": "689",
    "commit-sha": "ef9f102ba027100a65927ae925fcf0afb4018e03"
  },
  {
    "project": "kube_host",
    "component": "enforcerd",
    "pr-id": "915",
    "commit-sha": "9a9ad104135a007bb0b23b39e09e617559073963"
  }
]
```

**Review by dstiliadis — APPROVED** [review]

**Pull request merged**